code-server/README.md

```diff
@@ -79,3 +79,16 @@ module "code-server" {
   extensions = ["dracula-theme.theme-dracula", "ms-azuretools.vscode-docker"]
 }
 ```
+
+### Offline Mode
+
+Just run code-server in the background, don't fetch it from GitHub:
```

Comment:
Two thoughts here as I was trying out the module:

1. It could be helpful to indicate where to put code-server to make this work.  Here is a shot at some copy but it can probably be worded better:

```suggestion
Run an existing copy of code-server if found, otherwise download from GitHub. By default the module looks for code-server at `/tmp/code-server` but this can be changed with `install_prefix`.
```

2. Will users prefer if we throw an error when we cannot find code-server, given the `offline` name?  It could be surprising that `offline` could still pull from GitHub in some cases.  Or we could call this `use_existing` or `use_cached` instead or something like that, to avoid giving the impression it is fully offline.

Reply:
I was wondering if "offline" should be strict like you use and should fail if not present. 

And I can add a "use_cache" flag for a speed boost on restarts.

Reply:
Makes more sense to me.